src/utils/validateIdentifierInputs.ts

```diff
@@ -19,25 +19,21 @@ export const validateIdentifierInputs = (
   identifierInputs: IdentifierInput[]
 ): boolean => {
   for (const { value, validationRegex, textInputType } of identifierInputs) {
-    if (
-      textInputType === "PAYMENT_RECEIPT" &&
-      (!value ||
-        !isMatchRegex(
-          value,
-          validationRegex
-            ? validationRegex
-            : defaultPaymentReceiptValidationRegex
-        ))
-    ) {
-      throw new Error(ERROR_MESSAGE.INVALID_PAYMENT_RECEIPT_NUMBER);
```

Comment:
hmm actually this part is still required (can just remove the `!value` check), as invalid payment receipt will throw a specific error message, see `ERROR_MESSAGE.INVALID_PAYMENT_RECEIPT_NUMBER`

Reply:
ah okay, understand! yup i think this will work, should we include a test case to check for empty payment receipt fields?